modules/mig_with_percent/README.md

```diff
@@ -0,0 +1,13 @@
+# Managed Instance Group (MIG) with percent
+
+This module allows you to set the percentage ratio of second version of instance template on Managed Instance Group.
+
+## Usage
+
+See the [simple example](../../examples/mig_with_percent/simple) for a usage example.
+
+## Testing
```

Comment:
This heading should be dropped; testing is discussed in the root module README.

Reply:
Removed.